src/components/organisms/HomeIntro.module.css

```diff
@@ -1,83 +1,160 @@
 .introWrapper {
-  background-color: var(--color-primary);
+  transform: skewY(-9deg);
   overflow: hidden;
+  height: 850px;
+  background-color: var(--color-primary);
 }
 
 .intro {
-  position: relative;
-  padding: var(--spacer);
-  color: var(--brand-white);
   display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column-reverse;
+  transform: skewY(9deg);
+}
+
+.education {
+  display: flex;
+  flex-direction: column;
   flex-wrap: wrap;
-  align-items: flex-end;
+  padding: var(--spacer);
+  margin-top: 10%;
+  color: var(--brand-white);
+  text-align: center;
 }
 
-.intro h2 {
+.education h2 {
+  margin-top: 0;
   color: var(--brand-white);
-  margin-top: var(--spacer);
 }
 
-.left,
-.right {
-  padding: var(--spacer);
+.education h4 {
+  margin-bottom: 0;
+  color: var(--brand-cyan);
+}
+
+.playButtonWrapper {
+  position: relative;
+  top: -340px;
+  height: 300px;
+  width: 300px;
+  z-index: 2;
 }
 
-.left {
-  width: 100%;
+.playButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 300px;
+  height: 300px;
+  padding: 7px;
+  background-image: var(--gradient-purple-cyan);
+  border-style: solid;
+  border-color: transparent;
+  border-radius: 50%;
+  border-width: 0px;
+  animation: rot 7s linear infinite;
 }
-.right {
-  width: 100%;
+
+.circle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: var(--color-primary);
+  height: 286px;
+  width: 286px;
+  border-style: solid;
+  border-color: transparent;
+  border-radius: 50%;
+  border-width: 7px;
 }
-.spacer {
-  width: 0;
+
+@keyframes rot {
+  0% {
+    transform: rotate(0deg);
+  }
+  25% {
+    transform: rotate(90deg);
+  }
+  50% {
+    transform: rotate(180deg);
+  }
+  75% {
+    transform: rotate(270deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
```

Comment:
suggestion(non-blocking): should also work if you just write the final result `100% {
    transform: rotate(360deg);
  }`

Reply:
🤯

Reply:
🤯